src/axes/categoryAxis.ts

```diff
@@ -117,22 +120,101 @@ namespace Plottable.Axes {
     }
 
     protected _getTickValues() {
-      return this.getDownsampleInfo().domain;
+      return (this._scale as Scales.Category).getDownsampleInfo().domain;
```

Comment:
Should we just make the type of this._scale, Scales.Category? Any reason to leave it as Scale within categoryAxis?